### DISCUSSION TITLE
List of old/current features that need some love <3

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#3657] [General]
Here I will add items which need a review/change to be improved upon or fixed. Some are old/retired features which need to be addressed, others are current live features which need some extra love.

- Pirate Zone: New mechanic intended to replace Treasure Island ([Proposal by Blasta](https://github.com/sunflower-land/sunflower-land/discussions/3659) for inspiration)
- Potion House: Extra love needed to re-invigorate it. Possible quick/easy solutions are to add x multipliers (10x the price of 1 game with 10x the points reward), seasonal mutant crops and possible resource inputs vs coins (can be more expensive)
- Marine Marvels: The system could either be overhauled entirely or cgreatly reduce the chances. Possibly adding riddles/answers within the game for players to find out requirements etc. Too rare atm IMO, no Crimson Carps found in 3 months I believe. [Chances lowered for the new seasonal mecha marvel, more info to come]
- Flower Codex Quests: Similar to the fishing quest, would be nice to have something similar for flowers.
- Magic Mushrooms: Related to buds who have been holding out for a while. [Currently being worked on, stay tuned for more info]

Future features highly requested/sought after:

- Skill revamp: https://github.com/sunflower-land/sunflower-land/discussions/3419#discussioncomment-8510774
- New animals (cows/sheep?): https://github.com/sunflower-land/sunflower-land/discussions/2240

**Comment by Blasta-alasta** [comment]
Have some ideas here (not sure if any are good or not, but i've put 'em out there) for Pirate Zone.
(https://github.com/sunflower-land/sunflower-land/discussions/3659)
